Dockerfile.mcp

```diff
@@ -0,0 +1,9 @@
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static-debian12:nonroot@sha256:e8a4044e0b4ae4257efa45fc026c0bc30ad320d43bd4c1a7d5271bd241e386d0
+WORKDIR /
+COPY dist/distr-mcp /distr-mcp
+COPY dist/*.spdx.json /usr/local/share/sbom/
```

Comment:
This is only for the frontend sbom which is only needed for the main docker image
```suggestion
```

the sbom for the binary is generated by the docker build step automatically (since `sbom: true` is set)